### PR TITLE
Fix M1 segfault

### PR DESCRIPTION
**Pull request by simoncozens**
See #14.

### CODE REVIEW

File: src/python/ttfautohint/__init__.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ def __init__(self, path=None, **kwargs):
         version_string = lib.TTF_autohint_version_string().decode('ascii')
         self.version_string = version_string
 
+        lib.TTF_autohint.argtypes = [c_char_p]
```
Comment (anthrotype) [inline src/python/ttfautohint/__init__.py:63]: maybe add a comment with a link to our github issue and/or the python bug for future memory.

**Review by anthrotype — APPROVED** [review]
Thanks a lot for fixing this!

**Comment by anthrotype** [comment]
I'll cut a release after you're happy to merge this

**Comment by simoncozens** [comment]
Yes please!

**Pull request merged**